integrate/data-streams/overview/index.md

```diff
@@ -54,45 +71,71 @@ The `template` column type currently only supports the `jsonnet` templating lang
 
-***Warning**: Events sent to a receiver without any exports will be permanently lost. This is because data is buffered per export, not per receiver.*
+Same settings also exist for a sink. Use the [`GET /v1/branches/{branchId}/sources/{sourceId}/sinks/{sinkId}/settings`](https://stream.keboola.com/v1/documentation/#/configuration/GetSinkSettings) endpoint and [`PATCH /v1/branches/{branchId}/sources/{sourceId}/sinks/{sinkId}/settings`](https://stream.keboola.com/v1/documentation/#/configuration/PatchSinkSettings) endpoint in that case.
 
-The requests are asynchronous and create a task that must be completed before the receiver or export is ready to use. The task status can be checked using the [`GET /v1/receivers/{receiverId}/tasks/{taskId}`](https://buffer.keboola.com/v1/documentation/#/configuration/GetTask) endpoint.
+## Delivery guarantees
 
-Export tables are created if they do not exist. If they already exist, the schema defined by `export.columns` must match the existing schema. If the table schema is manually altered and it no longer matches, the upload from staging storage to the table will fail. The data is kept in the staging storage for up to 7 days during which you can recover any failures.
+Depending on your use-case you may need different delivery guarantees for your stream. Follow the guide-lines below to ensure the desired outcome.
 
-## Delete Receivers and Exports
+### At most once
 
-Receivers may be deleted using the [`DELETE /v1/receivers/{receiverId}`](https://buffer.keboola.com/v1/documentation/#/configuration/DeleteReceiver) endpoint. Exports may be deleted using the [`DELETE /v1/receivers/{receiverId}/exports/{exportId}`](https://buffer.keboola.com/v1/documentation/#/configuration/DeleteExport) endpoint.
+To ensure no record is delivered twice, make sure that the client doesn't have retries when sending the records. In this case it's beneficial to use the settings endpoints to set `"storage.level.local.encoding.sync.wait"` to `false` in order to increase the throughput.
 
-## Update Receivers and Exports
+### At least once
 
-A receiver may be updated using the [`PATCH /v1/receivers/{receiverId}`](https://buffer.keboola.com/v1/documentation/#/configuration/UpdateReceiver) endpoint. Exports maybe updated using the [`PATCH /v1/receivers/{receiverId}/exports/{exportId}`](https://buffer.keboola.com/v1/documentation/#/configuration/UpdateExport) endpoint.
+In order to have every record delivered at least once, the client needs to implement retries for sending the records. Also use the settings endpoints to make sure that `"storage.level.local.encoding.sync.wait"` is set to `true` (default behavior). Note that this settings guarantees that the record was written to the local disk. It may still be lost in case of a hardware failure of the disk since there is no replication at the moment.
```

Comment:
```suggestion
In order to have every record delivered at least once, the client needs to implement retries for sending the records. Also use the settings endpoints to make sure that `"storage.level.local.encoding.sync.wait"` is set to `true` (default behavior). Note that this settings guarantees that the record was written to the local disk. 
```
I would not add to documentation what can change. I think it is our decision what is the SLA of the service. ATM and have plan how would we like to increase SLA. Probably I would also remove the last sentence `Note that this settings guarantees that the record was written to the local disk. ` as it may change and we could do it differently. E.g store it in another database or what so ever, not on the disk (implementation detail).